### ISSUE TITLE
nvd3 1.7.1 no longer available on npm

### ISSUE THREAD

**Issue by acangiani** [novus/nvd3#1855]
Hi!

I tried to install the dependencies on my project but it seems that ```nvd3 1.7.2``` it's no longer available through npm. Did anyone had this problem? 

#### System version
```
Distributor ID:	Ubuntu
Description:	Ubuntu 16.10
Release:	16.10
Codename:	yakkety
```

#### package.json  
```
{
  "name": "app",
  "description": "",
  "version": "0.1.0",
  "dependencies": {
    "d3": "5.5",
    "nvd3": "1.7.1"
  },
}
```

```
$npm install
npm ERR! Linux 4.8.0-26-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install"
npm ERR! node v4.2.6
npm ERR! npm  v3.5.2

npm ERR! No compatible version found: nvd3@1.7.1
npm ERR! Valid install targets:
npm ERR! 1.8.4, 1.8.3, 1.8.2, 1.8.1, 1.1.15, 1.1.15-beta2, 1.1.15-beta1, 1.1.14, 1.1.13, 1.1.12, 1.1.10, 1.0.0, 0.9.0
npm ERR! 
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>
...
```

The Cloudfare links are still working:

[1.7.1-nv.d3.min.css](https://cdnjs.cloudflare.com/ajax/libs/nvd3/1.7.1/nv.d3.min.css)
[1.7.1-nv.d3.min.js](https://cdnjs.cloudflare.com/ajax/libs/nvd3/1.7.1/nv.d3.min.js)
```

Thanks!

**Comment by liquidpele** [comment]
1.7.1 was never on npmjs, just bower iirc.

**Comment by acangiani** [comment]
is there a reason for the gap? Only 1.7.1 and 1.20 don't seem to be published on npm.  Can they be added to npm?

**Comment by liquidpele** [comment]
Okay, I published 1.7.1 for you.    1.2x is very very old, so I won't push that one up.

**Comment by acangiani** [comment]
I appreciate this, thanks! 

This issue can be closed.
